airbyte-integrations/connectors/source-surveymonkey/source_surveymonkey/spec.json

```diff
@@ -17,16 +17,12 @@
       "credentials": {
         "type": "object",
         "title": "Authentication Type",
+        "description": "Authenticate over oAuth flow or provide token",
         "oneOf": [
```

Comment:
I think we can even go a step further and put the entire auth block in the root object. There's no reason really to have two auth mechanisms in a oneOf, since the connector always takes the access token. This should probably look like the FB marketing oauth spec. 

FWIW I don't think we should have merged this connector in this current state, see: https://github.com/airbytehq/airbyte/pull/7433#discussion_r738962844

Reply:
Done